app/views/entryPage.scala.html

```diff
@@ -0,0 +1,25 @@
+@(keys: List[String], form: DynamicForm, formAction: play.api.mvc.Call, submitButtonValue: String)
```

Comment:
The file name doesn't tell me what it contains. Maybe 'editableEntry.scala.html'